game/world/opcode_handling/handlers/inventory/DestroyItemHandler.py

```diff
@@ -22,5 +22,6 @@ def handle(world_session, reader):
                 return 0
 
             world_session.player_mgr.inventory.remove_item(bag, source_slot, True)
+            world_session.player_mgr.quest_manager.update_surrounding_quest_status()
```

Comment:
Should we check in other cases like Trading / Selling items? Or even completing quests that require items? :thinking: